src/main/java/org/isf/admission/gui/PatientFolderBrowser.java

```diff
@@ -890,7 +890,7 @@ public LabBrowserModel() {
 
 			try {
 				labList = labManager.getLaboratory(patient);
-				getOlderDate(labList, "examDate");
+				getOlderDate(labList, "createdDate");
```

Comment:
```suggestion
				getOlderDate(labList, "labDate");
```

Reply:
@mwithi `labDate` will only work if there is a new method is added to the `Laboratory` object.   Reflection can't find it because the `getter` method is called `getDate` and not `getLabDate`.

If one looks at the history of the `Laboratory` object you'll see that the old comment suggested using `createDate`:

<pre>
	/**
	 * use <code>getCreatedDate()</code> instead
	 * @deprecated 
	 */
	@Deprecated
	public LocalDate getExamDate() {
		return examDate;
	}
</pre>

Reply:
yes, this beacuse the createDate and examDate got time, while labDate did not.
now labDate has seconds and can be used

Reply:
Variable name changed here and method added in CORE repo [PR](https://github.com/informatici/openhospital-core/pull/856)